misk/src/main/kotlin/misk/web/MiskWebModule.kt

```diff
@@ -163,10 +164,22 @@ class MiskWebModule(
       .to<MetricsInterceptor.Factory>()
 
     newMultibinder<ConcurrencyLimiterFactory>()
-    if (!config.concurrency_limiter_disabled) {
+    if (!config.concurrency_limiter_disabled && config.concurrency_limiter?.enabled != false) {
```

Comment:
if someone chooses to use the new configuration and wants to use that to disable limiting, they may do so.

(Happy to revert if others disagree!)

Reply:
Could the log level also be included in your new config then?

Reply:
I wasn't quite sure how to handle backwards compatibility for that- would we want to make it nullable in the `ConcurrencyLimiterConfig` then fall back to the original? it looks like the only place it's being used is the in the interceptor factory, which i was somewhat reluctant to touch

Reply:
Yes, I'd make it nullable in the new one and fallback, then update service code to use new field, then delete the old field and make it non-null with a default parameter.

Reply:
So I added it to the new config- didn't make it nullable, but gave it the same default in the class definition and created a default `ConcurrencyLimiterConfig` within `WebConfig` that uses `concurrency_limiter_disabled` to set `disabled`